src/bloomfilter.c

```diff
@@ -72,14 +66,18 @@ BloomFilter *bloomfilter_Create_Mmap(size_t max_num_elem, double error_rate,
        By calling mbarray_Header, we copy that header data
        back into this BloomFilter object.
     */
-    if (mbarray_Header((char *)bf, array, sizeof(BloomFilter)) == NULL) {
+    if (mbarray_Header((char *)&bf->header, array, sizeof(Header)) == NULL) {
```

Comment:
Have you checked that structs `Header` and `Bloomfilter` are binary compatible ? If they aren't - Doesn't this mean that bloomfilters compiled with 0.3.16j aren't usable in 0.3.17j ?

Reply:
They are now compatible.